optimum/intel/ipex/modeling_base.py

```diff
@@ -523,7 +523,7 @@ def _prepare_past_key_values(self, input_ids):
                 ]
             )
             return past_key_values
-        elif model_type == "bloom":
+        elif model_type == "bloom" and is_transformers_version("<", "4.44"):
```

Comment:
just from curiosity, what changed in bloom in transformers 4.44?

Reply:
attention block modeling was changed for the sake of compatibility with the cache classes https://github.com/huggingface/transformers/pull/31445#pullrequestreview-2122697114

Reply:
@IlyasMoutawwakil what about the second part of my comment? What if user has old model converted with previous transformers version and new transformers installed, such case will not work now.

Reply:
is there a way to check the version used for exporting the model ?
if there is we can use the custom modeling or or raise an error asking to re export.

Reply:
yes, it is possible to check using rt_info:

```
from optimum.intel.openvino import OVModelForCausalLM
ov_model = OVModelForCausalLM.from_pretrained("bigscience/bloom-560m", export=True)
transformers_version = ov_model.model.rt_info["optimum"]["transformers_version"].value
```